hardhat/scripts/create-proposals.ts

```diff
@@ -1,63 +0,0 @@
-import { encodeEvmScript, encodeMetadata } from '../../src/logic/proposals/encoding';
```

Comment:
Why was this script removed?

Reply:
Sorry, forgot to mention this.

The "faker" dependency has a story behind it: https://fossa.com/blog/npm-packages-colors-faker-corrupted/
This script used it to create proposals, but I don't think it was really used. I decided it's easier to just remove it (now that its broken) instead of fixing it.

Reply:
Ok. I'm pretty sure I used this script when I had to test locally and was quite handy then (if I'm remembering correctly)